packages/udaru-core/lib/ops/authorizeOps.js

```diff
@@ -19,6 +19,20 @@ function badImplementationWrap (next) {
   }
 }
 
+function getContext (params) {
+  return {
+    udaru: {
+      userId: params.userId,
+      organizationId: params.organizationId
+    },
+    request: {
+      currentTime: new Date().toISOString(),
+      sourceIp: params.sourceIpAddress,
```

Comment:
in terms of validation, these can be null, so I would only add them to the request object if they're not null (I wouldn't let params through with null values to be evaluated in contexts)

Reply:
Yes i was thinking this, i'm not sure what it would evaluate to, but was thinking policies could have conditions where it does one thing if accessed with no ip address e.g. through core, vs via a serving API... maybe a separate context variable would be good for this purpose

e.g. request.package = udaru-core